ocs_ci/utility/ibmcloud.py

```diff
@@ -203,3 +205,193 @@ def add_deployment_dependencies():
+
+    def attach_volume(self, volume, node):
+        """
+        Attach volume to node on IBM Cloud.
+
+	Args:
+            volume (str): volume id.
+            node (str): worker node to attach.
+
+        """
+        logger.info(f"attach_volumes:{node[0].get()['metadata']['labels']['failure-domain.beta.kubernetes.io/zone']}")
+        providerId = node[0].get()['spec']['providerID']
+        clusterid = providerId.split("/")[5]
+        worker_id = node[0].get()['metadata']['labels']['ibm-cloud.kubernetes.io/worker-id']
+
+        logger.info(f"volume is : {volume}")
+
+        cmd = f"ibmcloud ks storage attachment  create --cluster {clusterid} --worker {worker_id} --volume {volume} --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        logger.info(f"attachment command output: {out}")
+
+
+    def detach_volume(self, volume, node=None, delete_from_backend=True):
+        """
+        Detach volume to node on IBM Cloud.
+
+	Args:
+            volume (str): volume id.
+            node (str): worker node to detach.
+	    delete_from_backend (bool): True for deleting the disk from the
+                storage backend, False otherwise
+
+        """
+        providerId = node[0].get()['spec']['providerID']
+        clusterid = providerId.split("/")[5]
+        worker_id = node[0].get()['metadata']['labels']['ibm-cloud.kubernetes.io/worker-id']
+
+        logger.info(f"volume is : {volume}")
+
+        cmd = f"ibmcloud is volume {volume} --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        if out["status"] == "available":
+            attachmentId = out["volume_attachments"][0]["id"]
+
+            cmd = f"ibmcloud ks storage attachment  rm --cluster {clusterid} --worker {worker_id} --attachment {attachmentId}"
+            out = run_cmd(cmd)
+            logger.info(f"detachment command output: {out}")
+
+    def get_node_by_attached_volume(self, volume):
+        """
+        Get the node by attached volume on IBM Cloud.
+
+	Args:
+            volume (str): volume id.
+
+        """
+        cmd = f"ibmcloud is volume {volume} --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        if not out["volume_attachments"]:
+            logger.info(f"volume is not attached to node")
+        else:
+            worker_id = out["volume_attachments"][0]["instance"]["name"]
+            logger.info(f"volume is  attached to node: {worker_id}")
+
+    def get_data_volumes(self):
+        """
+	Get volumes in IBM Cloud.
+
+	"""
+        logger.info(f"get data volumes")
+
+        cmd = f"ibmcloud is vols --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        vol_ids = []
+        for vols in out:
+            vol_ids.append(vols["id"])
+
+        logger.info(f"volume ids are : {vol_ids}")
+
+    def wait_for_volume_attach(self, volume):
+        """
+        checks volume is attached to node or not
```

Comment:
```suggestion
        Checks volume is attached to node or not.

```
There should be one empty line before `Args:`

Reply:
Done